assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -12,7 +12,8 @@
           ng-click="makeDownload(logs)">
           Raw <i class="fa fa-external-link"></i>
         </a>
-        <a ng-click="goChromeless()">
+        <a
```

Comment:
this is still going to conflict with steves change, get it to where there is no diff in this file